drivers/power_domain/power_domain_intel_adsp.c

```diff
@@ -87,8 +87,12 @@ static int pd_intel_adsp_pm_action(const struct device *dev, enum pm_device_acti
 
 static int pd_intel_adsp_init(const struct device *dev)
 {
-	pm_device_init_suspended(dev);
-	return pm_device_runtime_enable(dev);
+#ifdef CONFIG_PM_DEVICE
+	return pm_device_driver_init(dev, pd_intel_adsp_pm_action);
+#else
+	UNUSED(dev);
+	return 0;
+#endif
```

Comment:
The intended behavior of power domains is to be forced on by default when PM is not used, is this the case with this modification?